build/fuchsia/sdk.gni

```diff
@@ -101,10 +101,15 @@ template("_fuchsia_fidl_library") {
     ]
 
     outputs = [
-      "$target_gen_dir/$library_name_slashes/cpp/fidl.h",
-      "$target_gen_dir/$library_name_slashes/cpp/fidl.cc",
       "$target_gen_dir/$library_name_slashes/cpp/tables.c",
```

Comment:
Do you still want to include tables.c for this library?

Reply:
Yes based on https://fuchsia-review.googlesource.com/c/fuchsia/+/623061/13/scripts/sdk/gn/base/build/fidl_library.gni#192, but I'm not really sure why. Will check with Ian.